src/Umbraco.Web/Redirects/RedirectUrlManagementController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Web;
+using System.Web.Hosting;
+using System.Web.Http;
+using System.Xml;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Services;
+using Umbraco.Web.WebApi;
+using File = System.IO.File;
+
+namespace Umbraco.Web.Redirects
+{
+    public class RedirectUrlManagementController : UmbracoAuthorizedApiController
```

Comment:
Since this is an Umbraco controller, it needs the `[PluginController("UmbracoApi")]` attribute to be routed consistently with the rest of our controllers.